.npmignore

```diff
@@ -1 +1,7 @@
+**/*.ts
```

Comment:
I guess that's ok for now, but at some point we could set it up in the way that ts files are published and that they can be used directly

Reply:
You want to continue publish `.ts` files? In this case, I need to change config.
Currently we have 2 cases: 
- `ts` 
- `js` + `d.ts`

Reply:
actually there is a problem in package.json now. the `main` says `index.ts`

That should to be changed to:
```js
"main": "index.js",
"types": "index.d.ts",
```

Reply:
Yes. Actually. I missed it. Thanks.
I'm not sure, but looks like it worked for me even without this change :)

So let's go with compiled files? We need to decide

Reply:
I guess it worked for you because it was not published.
I guess we can go ahead with `index.js` and `index.d.ts` now that we compile it. it will be then compatible for ts and non-ts users